apps/portal/src/components/layout/ThemeSelector.tsx

```diff
@@ -31,7 +31,7 @@ export function ThemeSelector() {
       setTheme={setTheme}
       theme={theme}
     >
-      <Button variant="default" size="icon" onClick={() => setOpen(true)}>
+      <Button iconOnly onClick={() => setOpen(true)}>
```

Comment:
`iconOnly` prop will make the button look square. 
<img width="371" alt="image" src="https://github.com/user-attachments/assets/3bcd86dc-b245-4d0e-a5d7-dff53b7cdcf1" />

Reply:
are we removing `variant='icon'`?

Reply:
Yes @abhinavrastogi-harness

Reply:
IMO we should go with a separate IconButton component. let's discuss today.